etc/testing/circle/launch-published.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+# shellcheck disable=SC1090
+source "$(dirname "$0")/env.sh"
+
+# deploy object storage
+kubectl apply -f etc/testing/minio.yaml
+
+helm repo add pach https://helm.pachyderm.com
+helm repo update
+
+VERSION="2.0.4"
```

Comment:
for the meantime this version is hard-coded. The intention is that in the near future we will make this test run in a go script against a series of published versions